chirp/wxui/__main__.py

```diff
@@ -0,0 +1,4 @@
+"""Entry point for starting chirp via `python -m chirp.wxui`"""
+from . import chirpmain
+
+chirpmain()
```

Comment:
I don't understand why we need to do this (meaning you running it with `python3 -mchirp.wxui` in the desktop file) when we have `chirp` installed via `console_scripts`. Can't you just run `chirp`?

Reply:
Okay makes sense. Annoying, but fair enough :)

I'mma wait for @vishwin to look over this as he was pushing the `setup.py` to the current state.